modules/apps/frontend-data-set/frontend-data-set-api/src/main/java/com/liferay/frontend/data/set/model/FDSSortItemBuilder.java

```diff
@@ -12,6 +12,20 @@
  */
 public class FDSSortItemBuilder {
 
+	public static AfterActiveStep setActive(Boolean active) {
+		FDSSortItemStep fdsSortItemStep = new FDSSortItemStep();
+
+		return fdsSortItemStep.setActive(active);
+	}
```

Comment:
These steps are somewhat mind-bending :dizzy_face: 

I see they are an existing pattern in DXP, so :+1:

Reply:
Yes, it took me a while to check all this....